Add language client to connect to language server (#258)
metafacture/metafacture-playground: README.md

 Follow the instructions of Metafacture LSP if you want to start a local running language server (default websocket `ws://localhost:8080/ls`).
 The Metafacture Playground also runs without a language server. If you don't want to connect a language server, you should change ```(def ls-sockets {:flux "ws://localhost:8080/ls"})``` to ```(def ls-sockets {})``` in ```views.cljs```. The Playground also runs when not removing this connection information but will show errors when no language server is reachable via the given websocket address.

### Install Metafacture Core

This project depends on [metafacture-core](https://github.com/metafacture/metafacture-core), which is work in progress.
It is possible to display the current version of dependencies in the UI. To display the version, please read this [section](#show-dependency-versions-in-ui). This is especially reasonable when installing on a server.

> Installing Metafacture Core manually is only necessary, if you want to use a not published version.

Reviewer: ```suggestion
Installing Metafacture Core manually is only necessary if you want to use an upublished version.
```

#### Use Master/Main or other branch
If the master/main or another branch is used, the content of the file should contain a link to the commit like [https://github.com/metafacture/metafacture-fix/commit/b36fcb9](https://github.com/metafacture/metafacture-fix/commit/b36fcb9) (Please use the short hash link).

### Show hints

> When opening the playground in the browser the first time there is a hint how to use shortcuts. This hint can be dismissed and will be displayed when reloading the website. It's possible to add more hints if necessary (via metafacture_playground/db.cljs key :hints :content in the default-db). Do see changes the playground must be restartet.

Reviewer: ```suggestion
When opening the playground in the browser for the first time there is a hint on how to use shortcuts. This hint can be dismissed and will be displayed when reloading the website. It's possible to add more hints if necessary (via metafacture_playground/db.cljs key :hints :content in the default-db). To see changes the playground must be restarted.
```
